server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -207,7 +206,23 @@ private Tag renderApplicantRow(Account applicant) {
             StyleUtils.even(Styles.BG_GRAY_100))
         .with(renderInfoCell(applicant))
         .with(renderApplicantInfoCell(applicant))
-        .with(renderActionsCell(applicant));
+        .with(renderActionsCell(applicant))
+        .with(renderDOBCell(applicant));
+  }
+
+  private Tag renderDOBCell(Account applicant) {
+
+    Optional<Applicant> newestApplicant = applicant.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
```

Comment:
Not blocking: To avoid repeating the `td()` call and associated styles, could this instead be written as:

String applicantDOB = account.newestApplicant().map(applicant -> applicant.getApplicantData().getDob()).orElse("");